lib/jsdom/living/interfaces.js

```diff
@@ -17,6 +15,7 @@ const generatedInterfaces = {
   URLSearchParams: require("whatwg-url/webidl2js-wrapper").URLSearchParams,
 
   EventTarget: require("./generated/EventTarget"),
+  NodeFilter: require("./generated/NodeFilter"),
```

Comment:
Maybe this should instead be grouped with `NamedNodeMap`?

Reply:
I grouped it with `NodeIterator` and `TreeWalker`, which use `NodeFilter`.